src/map/battle.cpp

```diff
@@ -2263,7 +2263,7 @@ int64 battle_addmastery(map_session_data *sd,struct block_list *target,int64 dmg
 			damage += damage * 30 / 100;
 	}
 
-	if(type == 0)
+	if (type == 0)
```

Comment:
Space behind if is the automatic standard formatting we have to get to eventually, but if you don't want this to count as a change because it makes using Blame harder, I can revert this part.

Reply:
Its for blaming, because you didnt change anything here.

Reply:
Ok!